modules/serverless-inmemorychannel.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+//  * serverless/knative_eventing/serverless-channels.adoc
+
+[id="serverless-inmemorychannel_{context}"]
+= Using the default InMemoryChannel configuration
+
+InMemoryChannels are for development use only, and should not be used in a production environment.
+
+The following are limitations of InMemoryChannel Channels:
+
+* No event persistence is available. If a Pod goes down, events on that Pod will be lost.
+* InMemoryChannel Channels do not implement event ordering, so two Events that are received in the Channel at the same time may be delivered to a Subscriber in any order.
```

Comment:
s/may/might

>! may v
Use only to indicate permission. Otherwise, use “can” or “might.”

Reply:
Changed to can :+1: